File: 프로그래머스/0/181920. 카운트 업/카운트 업.java
class Solution {
    public int[] solution(int start_num, int end_num) {
        int count = 0;
        int a = 0;
        
        for(int i = start_num; i <= end_num; i++)
        {
            count++;
        }
        
         int[] answer = new int[count];
        
             
        for(int i = start_num; i <= end_num; i++)
        {
            answer[a] = i;
            a++;
        }
        
        
        return answer;
    }
}

File: 프로그래머스/0/181920. 카운트 업/카운트 업.cs
using System;
using System.Collections.Generic;

public class Solution {
    public int[] solution(int start_num, int end_num) {
        int[] answer = new int[] {};
        List<int> list = new List<int>();
        
        
        for(int i = start_num; i<=end_num; i++)
        {
            list.Add(i);
        }
        answer = list.ToArray();
        
        
        return answer;
    }
}